microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -93,8 +93,19 @@ Previously, you could not query for the etcd version included with {microshift-s
 //
 // This content will be added post-GA, as it is asynchronous content.
 
-//[id="microshift-4-15-networking"]
-//=== Networking
+[id="microshift-4-15-networking"]
+=== Networking
+
+[id="microshift-4-15-networking-doc-enhancements"]
+==== Networking documentation enhancements
+
+Various networking documentation improvements are now available in the {product-version}{product-title} release.
+//still don't know if I like this intro :soso-hand-emoji:
+
+This release includes extensive information on the OVN-Kubernetes Container Network Interface (CNI) plugin and describes its usage with {microshift-short}. You can also view the various networking customizations and features that are accessible in a {microshift-short} instance. This includes overviews of the supported networking components and services as well as outlines of the network topology. For more information, see xref:../microshift_networking/microshift-cni.adoc#microshift-cni[About the OVN-Kubernetes network plugin].
+
+The {microshift-short} documentation now includes procedures on how to audit exposed network ports and view the port log settings. The updated documentation can be viewed in xref:../microshift_networking/microshift-networking-settings.adoc#microshift-exposed-audit-ports_microshift-networking[Auditing exposed network ports].
+
```

Comment:
hmm, we need to get the inbound/outbound and network policies in here for this note to be really worthwhile